generator/tree.go

```diff
@@ -145,6 +145,12 @@ func generateConfigModule(cfg *ModuleConfig, node *Node, nameToNode map[string]*
 			if !ok {
 				return // Unsupported type.
 			}
+
+			// Unaccessible metrics should throw warning
+			if n.Access == "ACCESS_NOACCESS" || n.Access == "unknown" {
```

Comment:
Maybe make a new function `metricAccess()` that is similar to `metricType` that deals with access data.